packages/shared/src/lib/analytics.ts

```diff
@@ -11,6 +11,7 @@ export enum Origin {
   TagsSearch = 'tags search',
   RealTime = 'realtime',
   NonRealTime = 'nonrealtime',
+  ChangelogPopup = 'changelog popup',
```

Comment:
This should be added to analytics changes table in PR description. 🙏

Reply:
Thank you! Just did that, if you can take another look just to make sure I haven't messed up something there and have set the right values. 🙏